wca-guidelines.md

```diff
@@ -99,6 +99,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify as well. In this case, only one of the scramblers is required to sign the scorecard.
```

Comment:
```suggestion
- 4g+) [RECOMMENDATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify. In this case, only one of the scramblers is required to sign the scorecard.
```
To align with @Rouxles feedback on the alternate proposal [here](https://github.com/thewca/wca-regulations/pull/1267/files).